internal/target/repository.go

```diff
@@ -53,14 +53,15 @@ func NewRepository(r db.Reader, w db.Writer, kms *kms.Kms, opt ...Option) (*Repo
 }
 
 // LookupTarget will look up a target in the repository and return the target
-// with its host set ids.  If the target is not found, it will return nil, nil, nil.
+// with its host set ids and credential library ids.  If the target is not found,
+// it will return nil, nil, nil, nil.
 // No options are currently supported.
-func (r *Repository) LookupTarget(ctx context.Context, publicIdOrName string, opt ...Option) (Target, []*TargetSet, error) {
+func (r *Repository) LookupTarget(ctx context.Context, publicIdOrName string, opt ...Option) (Target, []*TargetSet, []*CredentialLibrary, error) {
```

Comment:
The naming convention between TargetSet and CredentialLibrary are inconsistent IMO.  To make them consistent CredentialLibrary should be renamed TargetLibrary, TargetSet should be renamed HostSet, or rename both to Library and Set.

Reply:
This was originally named `TargetCredentialLibrary` but @mgaffney changed it to `CredentialLibrary` to remove stutter.  I will probably update `TargetSet` to `HostSet`

Reply:
Lets chat with Mike tomorrow about best naming here, TargetSet is also the name of the view